frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -51,28 +63,35 @@ const InputForm = () => {
   const [jobs, setJobs] = useState<JobCategoryInterface[]>([
     { id: 0, name: "API Unavailable" },
   ]);
+  const [skills, setSkills] = useState<string[]>([]);
+  const [autoCompleteFocus, setAutoCompleteFocus] = useState<boolean>(false);
```

Comment:
echoing comment from below but I don't think line 67 is necessary

Reply:
removing